src/core_modules/capture-core/components/WidgetComment/CommentSection/CommentSection.js

```diff
@@ -102,7 +97,7 @@ const CommentSectionPlain = ({
 
     const CommentItem = ({ value, storedAt, createdBy }) => (
         <div data-test="comment-item" className={cx(classes.item)}>
-            <div className={classes.avatar} /> {/* TODO: add avatar */}
+            {/* TODO: add avatar */}
```

Comment:
Removed the avatar wrapper, leaving the `todo` there. It was adding some empty padding.